src/utils/uci_wrt.c

```diff
@@ -730,62 +762,61 @@ int uwrt_add_dhcp_pool(struct uctx *context, char *ifname, int vlanid,
 
   limit = (limit < start) ? 0 : (limit - start) + 1;
 
-  sprintf(property, "dhcp.%s%d=dhcp", ifname, vlanid);
+  sprintf(property, "dhcp.%s=dhcp", ifname);
   if (uwrt_set_property(context->uctx, property) < 0) {
     log_trace("uwrt_set_property fail for %s", property);
     return -1;
   }
 
-  sprintf(property, "dhcp.%s%d.interface=%s%d", ifname, vlanid, ifname, vlanid);
+  sprintf(property, "dhcp.%s.interface=%s", ifname, ifname);
   if (uwrt_set_property(context->uctx, property) < 0) {
     log_trace("uwrt_set_property fail for %s", property);
     return -1;
   }
 
-  sprintf(property, "dhcp.%s%d.networkid=br-%s%d", ifname, vlanid, ifname,
-          vlanid);
+  sprintf(property, "dhcp.%s.networkid=br-%s", ifname, ifname);
   if (uwrt_set_property(context->uctx, property) < 0) {
     log_trace("uwrt_set_property fail for %s", property);
     return -1;
   }
 
-  sprintf(property, "dhcp.%s%d.dhcpv4=server", ifname, vlanid);
+  sprintf(property, "dhcp.%s.dhcpv4=server", ifname);
   if (uwrt_set_property(context->uctx, property) < 0) {
     log_trace("uwrt_set_property fail for %s", property);
     return -1;
   }
 
-  sprintf(property, "dhcp.%s%d.instance=edgesec", ifname, vlanid);
+  sprintf(property, "dhcp.%s.instance=edgesec", ifname);
   if (uwrt_set_property(context->uctx, property) < 0) {
     log_trace("uwrt_set_property fail for %s", property);
     return -1;
   }
 
-  sprintf(property, "dhcp.%s%d.ignore=0", ifname, vlanid);
+  sprintf(property, "dhcp.%s.ignore=0", ifname);
   if (uwrt_set_property(context->uctx, property) < 0) {
     log_trace("uwrt_set_property fail for %s", property);
     return -1;
   }
 
-  sprintf(property, "dhcp.%s%d.force=1", ifname, vlanid);
+  sprintf(property, "dhcp.%s.force=1", ifname);
   if (uwrt_set_property(context->uctx, property) < 0) {
     log_trace("uwrt_set_property fail for %s", property);
     return -1;
   }
 
-  sprintf(property, "dhcp.%s%d.start=%d", ifname, vlanid, start);
+  sprintf(property, "dhcp.%s.start=%d", ifname, start);
```

Comment:
```suggestion
  snprintf(property, ARRAY_SIZE(property), "dhcp.%s.start=%d", ifname, start);
```

Reply:
`ARRAY_SIZE` - where is it defined?

Reply:
I found it in https://github.com/nqminds/edgesec/blob/302f4fd0e9c445037fcceca8aeaea16acef9a858/lib/libnetlink/include/utils.h#L267, but I just remember that in this repo, you normally use `STRLEN` (although maybe it's worth renaming it to ARRAY_LENGTH) https://github.com/nqminds/edgesec/blob/c9bd32a1868992cfe1ff87b013e8765f96d6f16a/src/utils/os.h#L45

Reply:
I changed all definitions.